content/docs/gke/pipelines/authentication-pipelines.md

```diff
@@ -5,53 +5,139 @@ weight = 50
+
+However, you must update existing pipelines that use the [use_gcp_secret kfp sdk operator](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.extensions.html#kfp.gcp.use_gcp_secret). Remove the `use_gcp_secret` usage to let your pipeline authenticate to Google Cloud using the default service account.
+
+## Securing the cluster with fine-grained GCP permission control
+
+### Workload Identity
+
+> Workload Identity is the recommended way for your GKE applications to consume services provided by Google APIs. You accomplish this by configuring a [Kubernetes service account](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) to act as a [Google service account](https://cloud.google.com/iam/docs/service-accounts). Any Pods running as the Kubernetes service account then use the Google service account to authenticate to cloud services.
 
 Referenced from [Workload Identity Documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity). Please read this doc for:
 
-* Detailed introduction about workload identity.
+* A detailed introduction to Workload Identity.
 * Instructions to enable it on your cluster.
 * Whether its limitations affect your adoption.
 
 #### Terminology
 
 This document distinguishes between [Kubernetes service accounts](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) (KSAs) and [Google service accounts](https://cloud.google.com/iam/docs/service-accounts) (GSAs). KSAs are Kubernetes resources, while GSAs are specific to Google Cloud. Other documentation usually refers to both of them as just "service accounts".
 
-#### Authoring pipelines to use workload identity
+#### Authoring pipelines to use Workload Identity
 
-Pipelines don't need any change to authenticate to GCP, it will use the GSA bound to `pipeline-runner` KSA transparently.
+Pipelines don't need any specific changes to authenticate to Google Cloud. With Workload Identity, pipelines run as the Google service account that is bound to the KSA.
 
-However, existing pipelines that use [use_gcp_secret kfp sdk operator](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.extensions.html#kfp.gcp.use_gcp_secret) need to remove `use_gcp_secret` usage to use the bound GSA.
-You can also continue to use `use_gcp_secret` in a cluster with workload identity enabled, but pipeline steps with `use_gcp_secret` will use the GSA corresponding to the secret provided.
+However, existing pipelines that use [use_gcp_secret kfp sdk operator](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.extensions.html#kfp.gcp.use_gcp_secret) need to remove the `use_gcp_secret` usage to use the bound GSA.
+You can also continue to use `use_gcp_secret` in a cluster with Workload Identity enabled, but pipeline steps with `use_gcp_secret` will use the GSA corresponding to the secret provided.
```

Comment:
If both KSA and use_gcp_secret are enabled, use_gcp_secret will take affects.

Reply:
updated